lib/std/event/group.zig

```diff
@@ -35,7 +35,7 @@ pub fn Group(comptime ReturnType: type) type {
             return Self{
                 .frame_stack = Stack.init(),
                 .alloc_stack = AllocStack.init(),
-                .lock = Lock.init(),
+                .lock = Lock.initLocked(),
```

Comment:
Looking at the commit which caused this a31d9f92f282a878836a3ecac5a48d5f4037868c, the lock is supposed to be initiated in an unlocked state:
```suggestion
                .lock = .{},
```

Also, could you apply the same change to `std.event.Locked` as mentioned in the issue?